Assets/Scripts/OriginalPlayerCharcter.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class OriginalPlayerCharcter : MonoBehaviour
+{
+    //private = camelCase public = PascalCase
+    [SerializeField]
+    private int lives = 3;
+
+    [SerializeField]
+    private string name = "Hector";
+    //if they both are the same type use commas
+    [SerializeField]
+    private float jumpHeight = 5, speed = 5;
```

Comment:
If the numbers are serialized, these are considered magic numbers. Please fix these.